Makefile

```diff
@@ -34,3 +34,20 @@ generate-mix32 :
 doctest :
 	perl -i -e 'while (<ARGV>) { print unless /package-id base-compat-\d+(\.\d+)*/; }' .ghc.environment.*
 	doctest src
+
+native:
+	nix-build -A native.splitmix
```

Comment:
This one is arguably not unneeded, given the normal cabal flow, mostly added out of completeness